features/base/participants/reducer.js

```diff
@@ -106,7 +106,11 @@ function participant(state, action) {
         return state;
 
     case PARTICIPANT_ROLE_CHANGED:
-        // TODO: check how actually roles change!!
```

Comment:
I think this is redundant action and reducer. Instead "PARTICIPANT_UPDATED" action and reducer can be used.